dapr-spring/dapr-spring-boot-autoconfigure/src/test/java/io/dapr/spring/boot/autoconfigure/client/DaprClientPropertiesTest.java

```diff
@@ -0,0 +1,76 @@
+package io.dapr.spring.boot.autoconfigure.client;
```

Comment:
One last thing, mind adding the header at the top of this file? Then lgtm and Ill approve and merge